price-estimator/src/filter.rs

```diff
@@ -287,35 +260,34 @@ async fn estimate_amounts_at_price_atoms<T>(
     }
 }
 
-async fn estimate_best_ask_price<T>(
+async fn estimate_best_ask_price(
     token_pair: TokenPair,
     _query: QueryParameters,
-    price_rounding_buffer: f64,
-    orderbook: Arc<Orderbook<T>>,
+    orderbook: Arc<Orderbook>,
 ) -> Result<impl Reply, Infallible> {
+    // TODO: Similar rounding buffer issue as above. Here we only have an exchange rate but the true
+    // exchange rate depends on the amounts.
```

Comment:
TODO

Reply:
Given that rounding went from a factor to a summand I think it's fine to not consider it for an epsilon amount.